### ISSUE TITLE
Add additional future auction start blocks to cache for auction schedule tool

### ISSUE THREAD

**Issue by alfarok** [w3f/polkadot-wiki#4080]
### Description

The [auction tool](https://wiki.polkadot.network/docs/learn-auction#auction-schedule) currently relies on a cache to provide quick estimates of when various blocks in the auction process will occur.  The tool is fully automated with the one exception that new auction start blocks must be added to the backing json files as they are determined by council.  

To the best of my knowledge there is no way to query all available auction start dates without checking every new block using `query.auctions.auctionInfo(BLOCK_NUMBER)` 

This task requires adding newly added auction start dates for Polkadot and Kusama to their respective json files [here](https://github.com/w3f/polkadot-wiki/tree/master/components/utilities/data).

New auctions can be added with the following format and the weekly GitHub action will take care of updating all future properties as they become available.  Only the `index` and `startBlock` values are required as shown below:

```json
{
    "index": AUCTION_NUMBER,
    "startBlock": AUCTION_START_BLOCK_NUMBER,
    "startHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
    "endPeriodBlock": null,
    "endPeriodHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
    "biddingEndsBlock": null,
    "biddingEndsHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
    "onboardStartBlock": null,
    "onboardStartHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
    "onboardEndBlock": null,
    "onboardEndHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
    "startDate": null,
    "endPeriodDate": null,
    "biddingEndsDate": null,
    "onboardStartDate": null,
    "onboardEndDate": null
}
```

Currently,

Kusama is updated until `auction #67` which is estimated to begin `Mon Dec 26 2022`

and

Polkadot is updated until `auction #41` which is estimated to begin `Sat Feb 25 2023`

**Comment by alfarok** [comment]
One manual strategy for retrieving this information is by using the [polkadot-js calendar](https://polkadot.js.org/apps/#/calendar).

Looking through the events you will notice some that contain the following formatting:

- `6:49` `#13,201,600` Execute named scheduled task auctlp10 [via Democracy/Dispatch](https://polkadot.js.org/apps/#/democracy/dispatch)

This means an auction is scheduled to begin on block `#13,201,600`, cross-referencing this data in the existing auction tool you can see this is auction `#34`.  

Unfortunately there is even less information available on future events than what already exists in the current cache.  Kusama has even less info available so it looks like another source is required.

I will continue adding alternative options here in the comments for comparison.  It may be worth looking into how Polkadot-JS is compiling this information to see if it can be produced programmatically through extrinsics.

**Comment by alfarok** [comment]
One area to investigate is the [Kusama/Polkadot Scheduler](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.polkadot.io#/scheduler).  We could potentially poll for scheduled auctions in the dispatch queue and check if they exist in the cache.  This could run on a weekly GitHub action.

**Comment by alfarok** [comment]
`api.query.scheduler.agenda()` seems to deliver the information displayed in PolkadotJS from above.

For example in this screenshot, block# 16,165,188 is upcoming Kusama auction `#67`:

![image](https://user-images.githubusercontent.com/13341935/211096880-d666373e-bfde-4b96-8ed5-45a67a9100de.png)

Decoding this information seems a bit more complicating but here is an example of how it is handled in PolkadotJS:

https://github.com/polkadot-js/apps/blob/master/packages/page-scheduler/src/Scheduler.tsx#L43

**Comment by alfarok** [comment]
Addressed in #4284
